addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -214,6 +222,15 @@ if (_id != -1) then {
                 };
             } call CBA_fnc_directCall;
         };
+
+        GVAR(playerEHInfo) pushBack ([{
```

Comment:
The handler is still always added even if noone is listening for it.
Seems to be fairly easy to just add it in L92 instead.

Reply:
We could do this since https://github.com/acemod/ACE3/commit/922774d972f304f1c6edc6d2b3c49725030a16b4. Unsure how worth it is though, ACE3 is already set to use it.